src/MicrosoftTeams.ts

```diff
@@ -551,12 +556,24 @@ namespace microsoftTeams {
             notifyFailure(reason?: string): void;
         }
 
+        export interface SaveResult {
+            /**
+             * Connector's webhook Url returned as arguments to settings.save event as part of user clicking on Save
+             */
+            webhookUrl?: string;
+        }
+
         /**
          * @private
          * Hide from docs, since this class is not directly used.
          */
         class SaveEventImpl implements SaveEvent {
             public notified: boolean = false;
+            public result: SaveResult = {};
```

Comment:
It's pointless to initialize the result here given that you are setting its value in the constructor.